tests/unit/specs/components/maprecorder/MapRecorderWin.spec.js

```diff
@@ -47,7 +47,7 @@ describe('maprecorder/MapRecorderWin.vue', () => {
       expect(vm.error).to.equal(false);
       // Supported codecs under chrome.
       expect(vm.mimeType).to.equal('video/webm');
-      expect(vm.mimeTypes.length).to.equal(2);
+      expect(vm.mimeTypes.length).to.equal(3);
```

Comment:
I was wondering whether this line should stay or be completely removed... the test is about default values... from a point of view, it is assigned when building the data object but on the other, it's value is computed and dependent of the browser the test is running on... so is it really a default value ?

I let you take the decision...

```suggestion
```